### PR TITLE
Feature/survey task project aggregate

### PR DESCRIPTION
**Pull request by gannetson**

### CODE REVIEW

File: bluebottle/surveys/adapters.py [hunk 1]
```diff
@@ -45,7 +45,8 @@ def parse_answers(self, data):
             match = question_re.match(key)
             if match:
                 question = match.group(1)
-                answers[question] = data[key]
+                if bool(data[key]):
```
Comment (eodolphi) [inline bluebottle/surveys/adapters.py]: Why cast to bool? Does'nt 
```
if data[key]:
```
work?

File: bluebottle/surveys/models.py [hunk 2]
```diff
@@ -101,6 +224,7 @@ class Response(models.Model):
     submitted = models.DateTimeField(null=True, blank=True)
     project = models.ForeignKey('projects.Project', null=True, blank=True)
     task = models.ForeignKey('tasks.Task', null=True, blank=True)
+    user_type = models.CharField(max_length=200, blank=True, null=True)
```
Comment (eodolphi) [inline bluebottle/surveys/models.py]: maybe we can be explicit about the possible values: (task_member, initiator, organization).

File: bluebottle/surveys/adapters.py (hunk 1 above)
Comment (gannetson) [inline bluebottle/surveys/adapters.py]: if {} => True
if bool({}) => False

File: bluebottle/surveys/models.py (hunk 2 above)
Comment (gannetson) [inline bluebottle/surveys/models.py]: True

**Pull request merged**